src/components/vacation-requests-table/vacation-requests-table.tsx

```diff
@@ -122,8 +150,17 @@ const VacationRequestsTable = ({
         isRowSelectable={() => !formOpen}
         initialState={{
           sorting: {
-            sortModel: [{ field: "updatedAt", sort: "desc" }]
-          }
+            sortModel: adminMode
+              ? [{ field: "updatedAt", sort: "asc" }]
+              : [{ field: "updatedAt", sort: "desc" }]
```

Comment:
```suggestion
            sortModel: [{ "updatedAt", sort: adminMode ? "asc" : "desc" }]
```

Reply:
Doesn't work: `Object literal may only specify known properties, and '"updatedAt"' does not exist in type 'GridSortItem'.ts(2353)`

Reply:
Sorry, there was a typo 
```suggestion
            sortModel: [{ field: "updatedAt", sort: "asc" }]
```